server/forms/form.spec.ts

```diff
@@ -130,13 +132,19 @@ describe('Form', () => {
             reason: 'likely',
           },
         },
+        sessionID: 'sessionID',
       })
 
       const form = await Form.initialize(request)
 
       form.persistData()
 
       expect(form.request.session.referralApplication).toEqual({ type: 'standard', reason: 'likely' })
+      expect(outputFile).toHaveBeenCalledWith(
+        `${__dirname}${savedSessionPath}sessionID.json`,
+        JSON.stringify(request.session.referralApplication),
+        'utf-8'
+      )
```

Comment:
Yep, good idea

Reply:
This could be clearer by having `MY-TOKEN` as a variable, then `join`-ing the variables in the expectation. Something like:

```typescript
const token = 'MY-TOKEN'

...

expect(outputFile).toHaveBeenCalledWith(
          `[__dirname, savedSessionPath, token '.json'].join(),
          JSON.stringify(request.session.referralApplication),
          'utf-8'
       )
```